tools/mkmapdata/mkmapdata.sh

```diff
@@ -509,17 +484,28 @@ until ((nfile>${#INGRID[*]})); do
       echo "Skipping creation of ${OUTFILE[nfile]} as fast mode is on so skipping large files in NetCDF4 format"
    else
 
-      cmd="$mpirun $ESMF_REGRID --ignore_unmapped -s ${INGRID[nfile]} "
-      cmd="$cmd -d $GRIDFILE -m conserve -w ${OUTFILE[nfile]}"
-      if [ $type = "regional" ]; then
-        cmd="$cmd --dst_regional"
+      PATH_PREFIX=/glade/work/slevis
```

Comment:
We should have this outside of a particular user's space. On cheyenne it could be under: /glade/tss/

Reply:
For now I changed these path assignments to be based off of `$dir` because this is already assigned at the top of the script. This way I avoid using `slevis` in the path.

I am open to changing to something like `/glade/tss/` later.